app/views/candidate_mailer/_new_flow_reference_received.text.erb

```diff
@@ -1,7 +1,7 @@
 Dear <%= @application_form.first_name %>
 
-<%= @reference.application_form.application_choices.pending_conditions.first.provider.name %> has received a refrence for you from <%= @reference.name%>.
+<%= @reference.application_form.application_choices.pending_conditions.first.provider.name %> has received a reference for you from <%= @reference.name%>.
 
-You can sign into your account to check the progress of your references requests and offer conditions.
+You can sign into your account to check the progress of your reference requests.
```

Comment:
Did we deliberately drop the " and offer conditions" line here?

Reply:
yeah – just following on from the discussion about this email also being used in the potential scenario of candidates receiving references post recruitment (but also happy to add it back in for now)